version.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/urfave/cli"
+)
+
+// version will be populated by the Makefile, read from
+// VERSION file of the source code.
+var version = ""
+
+// gitCommit will be the hash that the binary was built from
+// and will be populated by the Makefile
+var gitCommit = ""
+
+var versionCommand = cli.Command{
+	Name:        "version",
+	Usage:       "output the runtime version",
+	Description: `The version command outputs the runtime version.`,
+	Action:      printVersion,
+}
+
+func printVersion(context *cli.Context) (err error) {
+	if version == "" {
+		_, err = os.Stdout.WriteString("runC unknown\n")
```

Comment:
It's runC on http://runc.io/.  And runC sort of helps avoid #24.  But whatever, I can change it ;).